Mathlib/MeasureTheory/Constructions/BorelSpace/Order.lean

```diff
@@ -153,11 +153,13 @@ section PartialOrder
 
 variable [PartialOrder α] [OrderClosedTopology α] [SecondCountableTopology α] {a b : α}
 
-@[measurability]
 theorem measurableSet_le' : MeasurableSet { p : α × α | p.1 ≤ p.2 } :=
   OrderClosedTopology.isClosed_le'.measurableSet
 
-@[measurability]
+@[fun_prop]
+theorem measurable_le : Measurable fun p : α × α => p.1 ≤ p.2 :=
+  measurableSet_setOf.mp measurableSet_le'
```

Comment:
I know this is enough for `fun_prop` to work, but could you add the version where you compare two `α`-valued functions?